client/src/js/components/bhRenderOption.js

```diff
@@ -0,0 +1,32 @@
+angular.module('bhima.components')
+  .component('bhRenderOption', {
+    templateUrl : '/modules/templates/bhRenderOption.tmpl.html',
+    controller  : RenderOptionController,
+    bindings    : {
+      orientation         : '=', // two-way binding
+    },
+  });
+
+
+/**
+ * bhDateEditor Component
```

Comment:
You should probably remove all the comments about `bhDateEditor`.  They don't apply here.